deploy/clowdapp.yaml

```
@@ -4,6 +4,12 @@ kind: Template
 metadata:
   name: edge-api
 objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
```

Secret not defined correctly.

```suggestion
- apiVersion: v1
  kind: Secret
```